content/src/service/deployments/DeploymentManager.ts

```diff
@@ -80,11 +91,46 @@ export class DeploymentManager {
       pagination: {
         offset: curatedOffset,
         limit: curatedLimit,
-        moreData: moreData
+        moreData: moreData,
+        lastEntityId: options?.lastEntityId,
+        next: nextRelativePath
       }
     }
   }
 
+  private calculateNextRelativePath(options: DeploymentOptions | undefined, lastDeployment: Deployment): string {
+    const nextFilters = Object.assign({}, options?.filters)
+
+    const field = options?.sortBy?.field ?? SortingField.LOCAL_TIMESTAMP
+    const order = options?.sortBy?.order ?? SortingOrder.DESCENDING
+
+    if (field == SortingField.LOCAL_TIMESTAMP) {
+      if (order == SortingOrder.ASCENDING) {
+        nextFilters.fromLocalTimestamp = lastDeployment.auditInfo.localTimestamp
+      } else {
+        nextFilters.toLocalTimestamp = lastDeployment.auditInfo.localTimestamp
+      }
+    } else {
+      if (order == SortingOrder.ASCENDING) {
+        nextFilters.fromEntityTimestamp = lastDeployment.entityTimestamp
+      } else {
+        nextFilters.toEntityTimestamp = lastDeployment.entityTimestamp
+      }
+    }
+
+    const nextQueryParams = qs.stringify(
```

Comment:
Every time you need to create or parse a query string, you can use [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/URLSearchParams), it doesn't require a new package and the API is stable and as standard as `fetch`

```ts
qs = new URLSearchParams({key: values} | "?string=true")
newQueryString = '?' + qs.toString()
```

Reply:
This was not a request for change

Reply:
The main problem with `URLSearchParams` are multi params. 

If you have `deploymentId: string[]`, `URLSearchParams` would transform it into `deploymentId=id1,id2,id3`. But, our API uses the following format: `deploymentId=id1&deploymentId=id2&...`. 

That's why we use `{ arrayFormat: 'repeat' }` as an option for `qs.stringify`. Sadly, `URLSearchParams` doesn't have that option available